orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -118,7 +120,11 @@ func getMagmadMconfig(
 	if err != nil {
 		return nil, err
 	}
-
+	orc8rVersion, res := os.LookupEnv("VERSION_TAG")
+	if !res {
+		glog.V(1).Info("Couldn't get value for VERSION_TAG Env variable.")
+		orc8rVersion = "0.0.0"
```

Comment:
A `0.0.0` placeholder for a missing version works, but it looks close enough to being a valid version that I can see somebody being unsure if it's a placeholder or not. Switch this to a value that's completely unmistakeable as a placeholder, and specify that it's a possible value in the `mconfigs.proto` definition.

Reply:
what value do you suggest? 0.0.-1?